tests/Bridge/Symfony/Routing/RouterOperationPathResolverTest.php

```diff
@@ -41,12 +41,11 @@ public function testResolveOperationPath()
         $this->assertEquals('/foos', $operationPathResolver->resolveOperationPath('Foo', ['route_name' => 'foos'], OperationType::COLLECTION, 'get'));
     }
 
-    /**
-     * @expectedException \ApiPlatform\Core\Exception\InvalidArgumentException
-     * @expectedMessage Subresource operations are not supported by the RouterOperationPathResolver.
-     */
     public function testResolveOperationPathWithSubresource()
     {
+        $this->expectException(\ApiPlatform\Core\Exception\InvalidArgumentException::class);
```

Comment:
Should be imported instead of using the FQN. It's weird that PHP CS Fixer doesn't fix that automatically.

Reply:
Erg yeah....this is going to be annoying.

Reply:
@dunglas done.

Reply:
Because PHP-CS-Fixer does not have such a fixer. :laughing:

Reply:
The closest thing is: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.12.0/src/Fixer/Import/FullyQualifiedStrictTypesFixer.php

but it only deals with existing imports.

Reply:
already fixed :)